### PR TITLE
[cli] Add tracking for version

### PR DESCRIPTION
**Pull request by erikareads**
Adds an event that tracks which version of the CLI is being used.

### CODE REVIEW

File: packages/cli/test/unit/index.test.ts [hunk 1]
```diff
@@ -1,11 +1,38 @@
...
+import './test/mocks/matchers';
 
 import { Output } from '../../src/util/output';
 import { TelemetryEventStore } from '../../src/util/telemetry';
 import { TelemetryBaseClient } from '../../src/util/telemetry/base';
 
 describe('main', () => {
   describe('telemetry', () => {
+    describe('version', () => {
+      it('tracks version', () => {
```
Comment (EndangeredMassa) [inline packages/cli/test/unit/index.test.ts:33]: **suggestion (non-blocking):** ‏test the negative case as well

**Review by EndangeredMassa — DISMISSED** [review]

File: packages/node/test/unit/dev.test.ts [hunk 2]
```diff
@@ -117,7 +117,8 @@ async function withDevServer(
         { runningTimeout: 300 }
       ));
 
-    test('with `waitUntil` from context rejecting a promise ', () =>
```
Comment (trek) [inline packages/node/test/unit/dev.test.ts:120]: not blocking, but I think I'd generally prefer this workaround to be its own PR. But we can discuss this process on Monday!

**Review by trek — DISMISSED** [review]

**Review by trek — APPROVED** [review]

**Review by Thompson1985 — APPROVED** [review]
